src/Pages/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+import Header from '../Components/Header';
```

Comment:
one thing I like to do in my codebases is add comments that describe parts of a file. So in this case on line 3, I would add something like 
```
//-- Components
```
and on line 5 
```
//-- Page components
```